app/views/providers/application_merits_task/has_other_opponents/show.html.erb

```diff
@@ -31,12 +31,14 @@
                   row.with_cell(text: opponent.full_name)
                   row.with_cell(text: opponent_type_description(opponent))
                   row.with_cell do
-                    link_to_accessible(
-                      t("generic.change"),
-                      opponent_url(opponent.type, @legal_aid_application, opponent),
-                      class: "govuk-link change-link",
-                      suffix: opponent.full_name,
-                    )
+                    if opponent.ccms_opponent_id.blank?
+                      link_to_accessible(
```

Comment:
After today's review, should this have a 'better' change title, i.e. an aria tag or hidden text including the name of the org?

Reply:
Should clarify, I don't think this is a blocker, but I wanted to highlight that we should be sure we cover it when discussing/creating the Accessibility tickets :+1:

Reply:
turns out it does have an`aria-label="Change John boy"` which I think suffices for accessibility?! and the opponent.full_name is delegated to opposbale/organisation/individual which is "aliased" to organisation#name, so works for orgs too.